soapack/interfaces.py

```
@@ -223,10 +233,12 @@ def _get_mask_fname(self,patch,version):
             f = files[0]
             return f
         return self.config['mask_path'] + f
-            
-    def get_mask(self,patch,version=None):
+
+    def get_mask(self, season, patch, array, version=None, kind='binary_apod'):
         if version is None: version = self.config['default_mask_version']
-        return enmap.read_map(self._get_mask_fname(patch,version))
+        if kind is not "binary_apod": assert(None not in [season, array])
+        assert(kind in ['binary_apod', 'spf_holes', 'spf_no_holes']) 
+        return enmap.read_map(self._get_mask_fname(patch, season, array, version, kind))
```

Could you make all of season patch and array default to None, so that people can still call the default mask with just patch?

---

`get_mask` is updated.